build.yaml

```diff
@@ -24,4 +24,14 @@ build:
   - npm: install
   - npm: run-script ci
     graceful: true
+  - script: |
+      cd examples
+      npm install
+      npm install ../
```

Comment:
You are right! I've changed the dependency to point to local file in the package.json:
```
"dependencies": {
  "cassandra-driver": "file:../",
  // ...
}
```

It should not be required now.